src/chai-form.ts

```diff
@@ -341,15 +358,17 @@ export class ChaiForm extends LitElement {
   }
 
   override render() {
+debugger
```

Comment:
Remove this from the PR

Reply:
VS Code's tasks settings should automatically attach the debugger, so you can just add a breakpoint here if you need to without modifying the code.